packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -420,50 +438,54 @@ const chartSize = computed(() => drilldownChartSize(outputPanel.value));
 
 const selectedParameters = ref<string[]>(props.node.state.selectedParameters);
 const selectedVariables = ref<string[]>(props.node.state.selectedVariables);
+const selectedErrorVariables = ref<string[]>(props.node.state.selectedErrorVariables);
 
-let pyciemssMap: Record<string, string> = {};
+const pyciemssMap = ref<Record<string, string>>({});
 const preparedChartInputs = computed(() => {
 	const state = props.node.state;
 
 	if (!state.calibrationId) return null;
 
+	// FIXME: Hacky re-parse CSV with correct data types
+	let groundTruth: DataArray = [];
+	if (csvAsset.value) {
+		const csv = csvAsset.value.csv;
+		const csvRaw = csv.map((d) => d.join(',')).join('\n');
+		groundTruth = csvParse(csvRaw, autoType);
+	}
+
 	// Merge before/after for chart
 	const { result, resultSummary } = mergeResults(
-		runResult.value,
 		runResultPre.value,
-		runResultSummary.value,
-		runResultSummaryPre.value
+		runResult.value,
+		runResultSummaryPre.value,
+		runResultSummary.value
 	);
+	const error = getErrorData(groundTruth, result, mapping.value);
```

Comment:
Can use errorData.value here as its being defined on via `props.node.active watcher`

If we prefer utilizing this then we should consider removing errorData.value

Reply:
didn't notice that was there. I fixed it thanks!